src/main/java/org/ballerinalang/docgen/docs/html/HtmlDocumentWriter.java

```diff
@@ -91,14 +96,33 @@ public void write(Collection<BLangPackage> packages) throws IOException {
         if (BallerinaDocUtils.isDebugEnabled()) {
             out.println("Generating HTML API documentation...");
         }
-        Files.createDirectories(Paths.get(outputFilePath));
```

Comment:
I'm sorry Nirmal I didn't get you, the above statement creates new directories for outputFilePath if they do not exist.

BTW this pull request does not include any changes, this is just to rebasing changes in release-0.8.0-RC2 branch to master. Thanks!